config/tests/guest/libvirt/upstream.cfg

```diff
@@ -0,0 +1,251 @@
+include tests-shared.cfg
+username = root
+password = 123456
+main_vm = vm1
+vms = vm1
+nettype = bridge
+netdst=virbr0
+display = nographic
+take_regular_screendumps = no
+keep_screendumps_on_error = no
+keep_screendumps = no
+store_vm_register = no
+restore_image_after_testing=no
+vga=none
+virt_install_binary = /usr/bin/virt-install
+hvm_or_pv = hvm
+machine_type = pseries
+only bridge
+no xen, lxc, esx, ovmf
+#Filterout unwanted disk types
+no qed,qcow2v3,raw_dd,vmdk, usb2
+only no_virtio_rng
+only smp2
+only no_9p_export
+only no_pci_assignable
+only (image_backend=filesystem)
+create_vm_libvirt=yes
+kill_vm=yes
+kill_vm_libvirt=yes
+env_cleanup=yes
+mem=8192
+smp=8
+threads=1
+cores=8
+sockets=1
+vcpu_sockets=1
+vcpu_cores=8
+vcpu_threads=1
+setvcpus_max = 8
+vcpu_maxcpus = 8
+qemu_binary=/usr/share/avocado-plugins-vt/bin/qemu
+emulator_path=/usr/share/avocado-plugins-vt/bin/qemu
+kernel=/home/kvmci/linux/vmlinux
```

Comment:
can this be as dynamic input as path can be changed

Reply:
or either we can  have  readme

Reply:
this is more of a sample test config and this is matched with other layers of flow...

Reply:
ok sure